File: single-language-analysis/dataset-files/java/TestSmell090.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestSmell090 {

    @Test
    public void testEnharmonics() {
        assertTrue(ophis.FSHARP.in(ophis.GFLAT.enharmonics()));
        assertTrue(ophis.GFLAT.enharmonics(false, "chroma"));
        for (Chroma chroma : ophis.westernChromaSet) {
            for (Note enharmonicNote : chroma.enharmonics()) {
                assertEquals(chroma, enharmonicNote);
            }
        }
    }
}

File: single-language-analysis/dataset-files/python/test_smell_090.py
# Original URL: https://github.com/OphisMusic/ophis/blob/afd3f6ab9aab9de57eaf633caaf24d0e7b5c74ad/tests/test_tonus.py#L34-L39

def test_smell_090():
    assert ophis.FSHARP in ophis.GFLAT.enharmonics()
    assert ophis.GFLAT.enharmonics(False, "chroma")
    for chroma in ophis.western_chroma_set:
        for enharmonic_note in chroma.enharmonics():
            assert chroma == enharmonic_note